sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -878,7 +879,45 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
-  val USE_ARROW_OPT = conf("spark.rapids.arrowCopyOptimizationEnabled")
+  val CBO_ENABLED = conf("spark.rapids.sql.cbo.enabled")
```

Comment:
Nit: Since eventually these configs will become user-visible I'd like to avoid acronyms or odd terms the user may not be familiar with.  I'd suggest just using "optimizer" instead of "cbo", as the user may not know what "cbo" means and the fact that the optimizer is cost-based is arguably an implementation detail.